OxfordBodleian/Juel-Jensen/BDLaethd18.xml

```diff
@@ -37,18 +39,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <physDesc>
                         <bindingDesc>
                             <binding contemporary="true">
-                                <decoNote xml:id="b1">Two wooden boards covered with orange tooled leather (full cover); textile inlays.
+                                <decoNote xml:id="b1">Two <material key="wood">wooden</material> boards covered with orange tooled <material key="leather"/>.
                                 </decoNote>
                                 <decoNote xml:id="b2" type="bindingMaterial">
                                     <material key="wood"/>       
                                     <material key="leather"/>     
-                                    <material key="textile"/>                               
-                                </decoNote>
-                                <decoNote xml:id="b3" type="SewingStations">4</decoNote>
-                                <decoNote xml:id="b5">Blind tooled ornament on the front and back covers and on the inner side of the leather cover
-                                    (circles with dots, straight lines, interlaced line fillet, double crescents with dots, criss-cross, a cross). 
-                                    <!-- Description of the tooled ornamentation might need some revision -->
+                                    <material key="textile"/>    
                                 </decoNote>
+                                <decoNote xml:id="b3" type="Cover" color="orange">Full cover decorated with blind tooled ornament
+                                    (circles with dots, straight lines, interlaced line fillet, double crescents with dots, criss-cross, a cross). The ornamentation decorates both the
+                                   inner and outer sides of the cover.
+                                    <!-- Description of the tooled ornamentation might need some revision --></decoNote>
+                                <decoNote xml:id="b4" type="SewingStations">4</decoNote>
+                                <decoNote xml:id="b5" type="Spine">The stitches of four endband chains are sewn into the leather of the spine, two being close to the head and two to the tail.</decoNote>
+                                <decoNote xml:id="b6" type="Other">Textile inlays.</decoNote>
```

Comment:
You could use the keyword textileInlay

Reply:
the value is not allowed. According to the guidelines, the presence of a textile inlay or of a mirror can be encoded in `<decoNote type="Boards"/>`.